<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QDialog" name="Main_Window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Level-Headed</string>
  </property>
  <layout class="QVBoxLayout" name="layoutMain">
   <item>
    <layout class="QVBoxLayout" name="layoutBaseGame">
     <item>
      <widget class="QLabel" name="lblBaseGame">
       <property name="text">
        <string>Choose Your Base Game:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBaseGame">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutLevelGenerator">
     <item>
      <widget class="QLabel" name="lblLevelGenerator">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Choose Your Level Generator:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboLevelGenerator">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="middleSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnConfigureSettings">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Configure Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnGenerateGame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Generate Game!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBaseGame</tabstop>
  <tabstop>comboLevelGenerator</tabstop>
  <tabstop>btnGenerateGame</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
